cluster/addons/fluentd-elasticsearch/kibana-image/Makefile

```diff
@@ -14,7 +14,7 @@
 
 .PHONY:	build push
 
-TAG = v4.6.1
+TAG = v4.6.1-1
```

Comment:
Exactly. Moreover, kibana 5.X is intended to work with ES 5.X, which contains a lot of compatible changes.

I will update it one day, but it requires some testing and maybe e2e tests changes.